gifted_build/src/server/routes/test.js

```diff
@@ -1,8 +1,31 @@
+const knex = require('knex')({
+    client: 'pg',
+    connection: {
+      user: 'build',
+      host: 'localhost',
+      database: 'api',
+      password: 'password',
+      port: 5432
+    },
+    pool: { min: 0, max: 5 } // no logic to these numbers
+}); // Without this semicolon the program will break b/c Javascript has a silly parser.
 const express = require('express');
 const router = express.Router();
 
+router.get('/products/', (req, res) => {
```

Comment:
If we are using this file for the product routes, this should have a second identifier within the route, such as '/products/list_all' in order for use to possibly add more routes down the line.

Also we should rename this file from test.js to products.js